Src/FluentAssertions/Primitives/ReferenceTypeAssertions.cs

```diff
@@ -422,5 +422,5 @@ public AndConstraint<TAssertions> NotBeAssignableTo(Type type, string because =
 
     /// <inheritdoc/>
     public override bool Equals(object obj) =>
-        throw new NotSupportedException("Calling Equals on Assertion classes is not supported.");
+        throw new NotSupportedException("Equals is not part of Fluent Assertions. Did you mean Be() instead?");
```

Comment:
🔧 This class doesn't have a `Be` method. It only has `BeSameAs`.

Reply:
oops!